xml_converter/src/xml_converter.cpp

```diff
@@ -160,10 +183,24 @@ void process_data(
     // Read in all the protobin waypoint markerpacks
     for (size_t i = 0; i < input_waypoint_paths.size(); i++) {
         cout << "Loading waypoint pack " << input_waypoint_paths[i] << endl;
-        read_protobuf_file(
+        read_burrito_directory(
             input_waypoint_paths[i],
             &marker_categories,
             &parsed_pois);
+
+        if (output_split_waypoint_dir != "") {
+            move_supplementary_files(input_waypoint_paths[i], output_split_waypoint_dir);
+        }
```

Comment:
This is going to copy more than just the necessary files. Which we only know about when we call the write data function.